packages/zcli-themes/tests/functional/preview.test.ts

```diff
@@ -87,9 +87,9 @@ describe('themes:preview', function () {
 
         try {
           await PreviewCommand.run([baseThemePath])
-        } catch {
           expect(ctx.stdout).to.contain(`Validation error ${baseThemePath}/templates/home_page.hbs:10:6`)
           expect(ctx.stdout).to.contain("'searcsh' does not exist")
+        } catch {
```

Comment:
I see, so since you have a `try` `catch` above you can handle errors within the standard flow of the logic.  Nice! Maybe then you could remove the `try` `catch` `finally` block in your test code, and just run `nock.cleanAll()` as an after action?

Reply:
Thanks so much for the review, @token-cjg!

In this case we do need to assert within the `try` block as the command will exit with an error on the first run. 
`oclif` seems to exist the process asynchronously so we must assert on the `stdout` right after running the command  as `stdout` is already empty buy the time it falls into the `catch` block.

I moved the `nock.cleanAll()` to an `after` action as you suggested. Makes much more sense 👍🏼